### PR TITLE
Mark Book Status 50007

### PR DESCRIPTION
**Pull request by abdullahallmehedi**

### CODE REVIEW

**Comment by SSalekin** [comment]
ok

**Comment by noor-framgia** [comment]
ok

File: app/controllers/users_books_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class UsersBooksController < ApplicationController
+  def update
+    if users_book_params.has_key? :update_reading_status
```
Comment (dieunb) [inline app/controllers/users_books_controller.rb]: if not?

File: app/services/users_book_service.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      UsersBook.new(user_id: user_id, book_id: book_id)
+  end
+
+  def update_reading_status
+    if @users_book.new_record? || @users_book.unread?
+      @users_book.reading!
+    else
+      @users_book.read!
+    end
+    @users_book.save
```
Comment (dieunb) [inline app/services/users_book_service.rb]: no need

File: app/controllers/users_books_controller.rb (hunk 1 above)
Comment (abdullahallmehedi) [inline app/controllers/users_books_controller.rb]: then it is request for "favorite". Salekin san is doing add favorite task.

**Pull request merged**